_sass/kroxylicious.scss

```diff
@@ -2,10 +2,10 @@
 // Required
 @import "./bootstrap/scss/functions";
 
-@import url('https://fonts.googleapis.com/css?family=Josefin+Sans:300,400,700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Questrial&display=swap');
 
 $font-family-sans-serif:
-        "Josefin Sans", system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue",
+        "Questrial", system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue",
```

Comment:
Should `system-ui` and `-apple-system` be quoted? or are they magic SCSS things?

Reply:
They're magical CSS properties, not to worry. Google says it's fine 😉

Reply:
/me moves on quietly